test/models/announcement_test.rb

```diff
@@ -1,37 +1,37 @@
-require 'test_helper'
+require "test_helper"
 
 class AnnouncementTest < ActiveSupport::TestCase
-  test 'can create an announcement with the proper info' do
-    assert_difference 'Announcement.count', +1 do
-      announcement_params = {title: 'NO School', body: 'WOOT WOOT', expiration_date: 2.days.from_now, created_by: 'richard'}
+  test "can create an announcement with the proper info" do
+    assert_difference "Announcement.count", +1 do
+      announcement_params = {title: "NO School", body: "WOOT WOOT", expiration_date: 2.days.from_now, created_by: "richard"}
       Announcement.create(announcement_params)
     end
   end
 
-  test 'can not create an announcement without title' do
-    assert_no_difference 'Announcement.count' do
-      announcement_params = {body: 'WOOT WOOT', expiration_date: 2.days.from_now, created_by: 'richard'}
+  test "can not create an announcement without title" do
+    assert_no_difference "Announcement.count" do
+      announcement_params = {body: "WOOT WOOT", expiration_date: 2.days.from_now, created_by: "richard"}
       Announcement.create(announcement_params)
     end
   end
 
-  test 'can not create an announcement without body' do
-    assert_no_difference 'Announcement.count' do
-      announcement_params = {title: 'NO School', expiration_date: 2.days.from_now, created_by: 'richard'}
+  test "can not create an announcement without body" do
+    assert_no_difference "Announcement.count" do
+      announcement_params = {title: "NO School", expiration_date: 2.days.from_now, created_by: "richard"}
```

Comment:
Line is too long. [105/80]<br>Space inside { missing.<br>Space inside } missing.